config/app.php

```diff
@@ -13,7 +13,7 @@
     |
     */
 
-    'name' => env('APP_NAME', 'Laravel'),
+    'name' => env('APP_CURRENT_NAME', env('APP_NAME', 'Laravel')),
```

Comment:
Why do we need a new env variable for this?

Reply:
Unfortunately, `APP_NAME` cannot be changed on fortrabbit :(